velox/dwio/common/tests/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 add_subdirectory(utils)
+velox_add_library(velox_dwio_faulty_file_sink FaultyFileSink.cpp)
+velox_link_libraries(velox_dwio_faulty_file_sink velox_file_test_utils
```

Comment:
This is a test utility right? In that case it should maybe live in `utils` but it should def use the base cmake functions `add_library` and `target_link_libraries`  to create a standalone lib, that way when we can install velox you can build with test stuff but the actual library is still 'clean'.